src/core/Akka/Actor/SupervisorStrategy.cs

```diff
@@ -350,7 +350,8 @@ public OneForOneStrategy(int maxNrOfRetries, int withinTimeMilliseconds, IDecide
         /// Constructor that accepts only a decider and uses reasonable defaults for the other settings
         /// </summary>
         /// <param name="localOnlyDecider">TBD</param>
-        public OneForOneStrategy(Func<Exception, Directive> localOnlyDecider) : this(-1, -1, localOnlyDecider, true)
+        /// <param name="loggingEnabled">TBD</param>
+        public OneForOneStrategy(Func<Exception, Directive> localOnlyDecider, bool loggingEnabled = true) : this(-1, -1, localOnlyDecider, loggingEnabled)
```

Comment:
This change triggers our API Approval test ;) 
In approve this change. With one addendum: Describe in the parameter comments what it does and what it was added. Does not have to be very extensive. But it needs to be clear for a user what it does and why its there.